src/main/java/org/openelisglobal/common/rest/provider/PatientSearchRestController.java

```diff
@@ -138,4 +164,126 @@ private PatientSearchWorker getAppropriateWorker(HttpServletRequest request, boo
                 externalID, patientID, guid, dateOfBirth, gender);
     }
 
+    private List<org.hl7.fhir.r4.model.Patient> parseCRPatientSearchResults(Bundle patientBundle) {
+        return patientBundle.getEntry().stream().filter(entry -> entry.hasType("Patient"))
+                .map(entry -> (org.hl7.fhir.r4.model.Patient) entry.getResource()).collect(Collectors.toList());
+    }
+
+    public Patient transformFhirPatientObjectToOpenElisPatientObject(Patient openELISPatient,
```

Comment:
MOve this method to the T[ransformService](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirTransformServiceImpl.java)

Reply:
Yeah sure!

Reply:
i will transfer stuff in my next commit